Marlin/configuration_store.cpp

```diff
@@ -64,28 +64,31 @@
  *  195  M206 XYZ  home_offset                      (float x3)
  *  207  M218 XYZ  hotend_offset                    (float x3 per additional hotend)
  *
- * Global Leveling:
+ * Global Leveling:                                 6 bytes
  *  219            z_fade_height                    (float)
+ *  223            GRID_MAX_POINTS_X                (uint8_t)
+ *  224            GRID_MAX_POINTS_Y                (uint8_t)
```

Comment:
Sure, but what's the logic there?  I couldn't come up with a good reason to have multiple, as we'll never have multiple bed levelings active at the same time, and they're all using the same #DEFINE, so any change to grid size will necessarily ruin the others anyway.